docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,36 @@
+# Participants onboarding in MVD
+
+This document outlines the infrastructure steps for the scenario of onboarding participants to the Minimum Viable Dataspace (MVD).
+
+## Precondition - GAIA-X membership
+
+To join the MVD, each participant must have a Verifiable Credential (VC) signed by GAIA-X Authority which proves GAIA-X membership.
+
+At present the MVD deployment pipeline does not interact with the real GAIA-X Authority. A simulated GAIA-X Authority is deployed within MVD for demonstration purposes.
+
+## Onboarding process - GAIA-X membership verification
+
+During the onboarding process, the GAIA-X membership claim is verified by the Dataspace Authority.
+The onboarding flow is presented in the [distributed authorisation sub-flow document](https://github.com/agera-edc/MinimumViableDataspace/tree/feature/20-rs-adr-target/docs/developer/decision-records/2022-06-16-distributed-authorization).
+Actors in the onboarding scenario:
+
+- _Participant A_ is a putative participant that wants to join the MVD.
+- _Participant B_ is the Dataspace Authority (Registration Service and its `did:web` document).
+- _Authority_ is the simulated GAIA-X Authority.
+
+## Infrastructure
+
+To enable above precondition and onboarding scenario in MVD, the following steps need to be implemented in the MVD infrastructure: 
+
+1. The MVD deployment workflow will generate the private and public keys for GAIA-X Authority.
+2. The MVD deployment workflow will deploy the GAIA-X Authority DID document containing the public key.
+3. The GAIA-X private key will be used in the participant deployment workflow to generate GAIA-X membership Verifiable Credentials. Additionally, 
+   it will be published as Github artifact so that it can be used locally to onboard additional participants.
```

Comment:
Publishing private key as artifact will expose the key, right ? Have you considered storing this key in key vault and from there this can be retrieved manually if needed ? 

What do you suggest ? Thanks

Reply:
I think it's the easiest approach, and this is not a production system. But, we don't need to delay the adoption of the ADR determining the overall design until we decide this point of detail. So we could rewrite this in a more general way and decide later. e.g. "Additionally, it will be made available to be used locally to onboard additional participants."

Reply:
okay we can write in a more genera way then and defer the decision. Thanks.